recipe/recipe.yaml

```diff
@@ -0,0 +1,57 @@
+context:
+  name: libusb
+  version: 1.0.28
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/libusb/libusb/releases/download/v${{ version }}/${{ name }}-${{ version }}.tar.bz2
+  sha256: 966bb0d231f94a474eaae2e67da5ec844d3527a1f386456394ff432580634b29
+
+build:
+  number: 0
+
+requirements:
+  run_exports:
+    - ${{ pin_subpackage(name) }}
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    # cmake is only for doing string substitution for creating pkg-config file
+    - if: win
+      then:
+        - cmake
+    - if: unix
+      then:
+        - libtool
+        - make
+        - gnuconfig
+        - automake 1.17.*
```

Comment:
Is there a reason why automake 1.17 is required specifically, or just protecting against potential incompatibilities that break the build in the future?

Reply:
I had to add 1.17 specifically otherwise the build failed. The logs have been cleaned unfortunately. I dont remember from the top of my head what the reason was. We can try to remove it and see what happens?

Reply:
No that's fine, it's now at least documented in these comments that it is necessary and we can investigate further if it ever becomes an issue down the line.